src/stories/components/CioPlp/CioPlp.stories.tsx

```diff
@@ -61,6 +61,42 @@ const meta = {
         },
       },
     },
+    paginationConfigs: {
+      description:
+        'Configuration options for pagination. Used to set windowSize of pages array and can override resultsPerPage.',
+      table: {
+        type: {
+          summary: 'UsePaginationProps',
+        },
+      },
+    },
+    sortConfigs: {
+      description: 'Configuration options for sorting functionality.',
+      table: {
+        type: {
+          summary: 'UseSortProps',
+        },
+      },
+    },
+    filterConfigs: {
+      description: 'Configuration options for filters functionality.',
+      table: {
+        type: {
+          summary: 'UseFilterProps',
+        },
+      },
+    },
+    groupsConfigs: {
+      description:
+        'Configuration options for the Groups component. Controls initialNumOptions, isCollapsed, title, and hideGroups.',
+      table: {
+        type: {
+          summary: 'GroupsProps',
+        },
+        defaultValue: { summary: '{}' },
+      },
+      control: { type: 'object' },
+    },
```

Comment:
Let's not add documentation this way. This overrides the JSDocs you've written as part of `UseCioPlpProps` which is preferred.